### ISSUE TITLE
An item with the same key has already been added. Key: notify_customer

### ISSUE THREAD

**Issue by ajayak** [nozzlegear/ShopifySharp#356]
# Issue
Issue when setting `NotifyCustomer` property while creating or updating fulfilment.

## Possible resolution
Remove the redundant parameter **bool notifyCustomer** from `FulfillmentService` `CreateAsync` and `UpdateAsync` function as it is already available in **Fulfillment fulfillment**

## Steps to reproduce

```cs
var fulfillment = await fulfillmentService.CreateAsync(orderId, inputFulfillment, true);
```

**Comment by nozzlegear** [comment]
Thanks for reporting this! I've got some free time in the next couple of days and will get it fixed.

**Comment by ajayak** [comment]
Hi @nozzlegear, any update on this? 😃

**Comment by nozzlegear** [comment]
This is fixed in 4.20.0 on Nuget!
